clang/test/Sema/compound-literal.c

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsyntax-only -verify -fblocks -pedantic %s
+// RUN: %clang_cc1 -fsyntax-only -verify -fblocks -pedantic -Wno-comment %s
```

Comment:
Why did this test need to add `-Wno-comment`?

Reply:
[This line](https://github.com/llvm/llvm-project/pull/91891/files#diff-7d0182b6392ee774cf192fde090a22efd8367b693b28bac42d2e3ae2f8c2d94bR50-R51) both warns about the empty initializer being used without `-std=c2x`, and now triggers the new diagnostic. As I understand `-verify`, this cannot be tested using `//` on subsequent lines, so I used the `\` syntax to chain these checks, as some other tests do. Without `-Wno-comment`, this would lead to an additional warning for the multiline `// ... \` comment.

Reply:
Ah, that's good to know! Seeing as I had to modify the test again anyway, it seems nicer to use this syntax.